projects/1.16/assets/minecraftforge/forge/lang/zh_cn.json

```
@@ -0,0 +1,170 @@
+  "forge.configgui.zombieBabyChance.tooltip": "小僵尸在所有僵尸（包括僵尸的子类）中的出现概率。更改此项可以修改僵尸生成机制。",
+  "forge.configgui.zombieBabyChance": "小僵尸概率",
+  "forge.configgui.logCascadingWorldGeneration.tooltip": "日志记录生成层叠区块时的地形填充问题。",
+  "forge.configgui.logCascadingWorldGeneration": "日志记录层叠世界生成",
+  "forge.configgui.fixVanillaCascading.tooltip": "修复原版层叠世界生成的问题。这会改变原版世界生成器的运作方式，请不要反馈任何有关与原版世界生成不一致的现象。",
+  "forge.configgui.fixVanillaCascading": "修复原版层叠世界生成问题",
+  "forge.configgui.dimensionUnloadQueueDelay.tooltip": "服务端在卸载某个维度前的以刻为单位的等待时间。在快速加载或卸载维度的情况下（例如向下界传送门中按一定频率丢物品，迫使其保持加载），适当调整此选项会有所帮助。",
+  "forge.configgui.dimensionUnloadQueueDelay": "维度卸载延时",
+  "forge.configgui.clumpingThreshold.tooltip": "调整 Packet51 优于 Packet52 的临界值，默认最小为 64、最多为 1024。",
+  "forge.configgui.clumpingThreshold": "数据包成簇临界值",
+  "forge.configgui.treatEmptyTagsAsAir.tooltip": "原版将使用空标签的处理配方，且并不允许你合成该槽位中的任何内容。如果该选项为 false，则此项更改中的空标签会使用屏障作为物品。目的是防止使用空气合成。",
+  "forge.configgui.treatEmptyTagsAsAir": "将空标签作为空气处理",
+  "forge.configgui.zoomInMissingModelTextInGui.tooltip": "关闭此项会使 GUI 的缺失模型文本适配栏位。",
+  "forge.configgui.zoomInMissingModelTextInGui": "放大 GUI 中的模型缺失文本",
+  "forge.configgui.forgeCloudsEnabled.tooltip": "将云的渲染交由 GPU 处理以获得更快的渲染速度。",
+  "forge.configgui.forgeCloudsEnabled": "使用 Forge 云渲染器",
+  "forge.configgui.disableStairSlabCulling.tooltip": "禁用剔除楼梯和台阶旁边的隐藏面。这样会造成额外渲染，但可能会修复一些利用此原版机制的资源包。",
+  "forge.configgui.disableStairSlabCulling": "禁用楼梯/台阶剔除",
+  "forge.configgui.alwaysSetupTerrainOffThread.tooltip": "启用后，Forge 会将所有区块更新分配到独立的区块更新线程。\n可能会大幅提升 FPS，但也可能产生奇怪的渲染卡顿。\n不推荐在可用核心数低的电脑上启用此选项。",
+  "forge.configgui.alwaysSetupTerrainOffThread": "强制线程化区块渲染",
+  "forge.configgui.forgeLightPipelineEnabled.tooltip": "启用 Forge 方块渲染管线 —— 修复自定义模型的光照问题。",
+  "forge.configgui.forgeLightPipelineEnabled": "Forge 光照管线已启用",
+  "forge.configgui.selectiveResourceReloadEnabled.tooltip": "启用后，特定的重载任务（如切换语言）会更快进行。",
+  "forge.configgui.selectiveResourceReloadEnabled": "启用选择性资源加载",
+  "forge.configgui.showLoadWarnings.tooltip": "启用后，Forge 将显示加载过程中出现的任何警告。",
+  "forge.configgui.showLoadWarnings": "显示加载警告",
+  "forge.configgui.disableVersionCheck.tooltip": "该项为 true 时会禁用 Forge 版本检测机制。Forge 会在我们的服务端上对比查询一个较小的 json 文件以获取版本信息。更多细节请参阅在 GitHub 上的 ForgeVersion 类。",
+  "forge.configgui.disableVersionCheck": "禁用 Forge 版本检测",
+  "forge.controlsgui.shift": "SHIFT + %s",
+  "forge.controlsgui.control": "CTRL + %s",
+  "forge.controlsgui.control.mac": "CMD + %s",
+  "forge.controlsgui.alt": "ALT + %s",
+  "forge.container.enchant.limitedEnchantability": "附魔能力限制",
+  "attribute.name.forge.swim_speed": "游泳速度",
+  "attribute.name.forge.nametag_distance": "名称标签渲染距离",
+  "attribute.name.forge.entity_gravity": "实体重力",
+  "attribute.name.generic.reach_distance": "触及距离",
+  "forge.swimSpeed": "游泳速度",
+  "forge.nameTagDistance": "名称标签渲染距离",
+  "forge.entity_gravity": "实体重力",
+  "generic.reachDistance": "触及距离",
+  "fluid.minecraft.milk": "奶",
+  "fluid.minecraft.flowing_milk": "奶"
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "fluid.minecraft.milk": "牛奶",
  "fluid.minecraft.flowing_milk": "牛奶"
```

---

1.16 实装了 1.17 的译名改动（建议更新 index，如果没看到变动那开个新游戏实例）

---

！可是为什么要这么做呢，是 CFPA 的决定吗

---

原版是什么这就是什么，是官中的决定